app/assets/javascripts/covid19/src/webapp/components/structures-table/Columns.tsx

```diff
@@ -174,6 +193,8 @@ export function getColumns(
                                 data={data}
                                 onClickDetails={options.onClickDetails}
                                 onClickIDR={options.onClickIDR}
+                                onClickNMR={options.onClickNMR}
+                                setNMROptions={options.setNMROptions}
                                 validationSources={data.validationSources}
```

Comment:
No need to change, just a as a not, when prop names match as in this case: `{...options}`